third_party/gems/vendor_cache.BUILD

```diff
@@ -0,0 +1,14 @@
+
+filegroup(
+    name = "cache",
+    srcs = [ %{gemfiles} ],
```

Comment:
I didn't understand what this does, could you leave comments?

Reply:
I should have removed this template, thanks for catching this.